### DISCUSSION TITLE
some maps have issues in beatmap select screen

### DISCUSSION THREAD

**Discussion by shinmorisawa** [ppy/osu#30941] [Q&A] [answered]
### Type

Game behaviour

### Bug description

the artist name (camellia in this case) is bugged out. it looks like unicode if it was converted to ascii.
note: i am using linux, but i don't think this is a linux problem.

### Screenshots or videos

![osu_2024-12-01_20-48-11](https://github.com/user-attachments/assets/999c8204-eff4-48ff-9d44-63e2a0a666e2)
![osu_2024-12-01_20-48-28](https://github.com/user-attachments/assets/a928e519-9f05-4c21-9b36-0425c2276d05)

### Version

2024.1115.3

### Logs

not necessary

**Comment by Joehuu** [comment, marked as answer]
Relevant beatmap: https://osu.ppy.sh/beatmapsets/675779#osu/1431734

This is a problem with the beatmap itself. In `Camellia - NUCLEAR-STAR (ProfessionalBox) [Sing's Hard].osu`:
```
ArtistUnicode:ã‹ã‚ã‚Šã‚
```

Converting to discussion as it is not an osu! issue.

**Reply by shinmorisawa** [reply]
ok thanks.
